README.md

```diff
@@ -132,6 +92,184 @@ For instance, mobile app devs might be interested mainly in
 While backend developers might want to directly use `lwk_cli`
 in their systems.
 
+## Getting started with LWK Development
+
+### Rust
+
+#### Rust Examples
+
+* [Create a testnet watch-only wallet from a CT wallet descriptor and get list of transactions](./lwk_wollet/examples/list_transactions.rs)
+
+### Python
+
+#### Build Python wheel
```

Comment:
you can also install `lwk` from Pypi with `pip install lwk`, I don't know if we want to mention it here